addons/website_sale/controllers/main.py

```diff
@@ -1001,6 +1002,12 @@ def confirm_order(self, **post):
 
         return request.redirect("/shop/payment")
 
+    def _update_so_external_taxes(self, order):
+        try:
+            order.validate_taxes_on_sales_order()
+        except ValidationError:
+            pass
```

Comment:
Comment here too why you're silencing the exception

Reply:
Done